packages/plugins/src/foundry/decode.ts

```diff
@@ -92,127 +95,21 @@ export const decodeDeploymentInfo = (
-    )
-  }
-
-  const actionInputs: Array<RawActionInput> = []
-  for (const {
-    transaction,
-    transactionType,
-    additionalContracts,
-  } of dryRun.transactions) {
-    if (transaction.value !== undefined && transaction.value !== '0x0') {
-      console.error(
-        `Sphinx does not support sending ETH during deployments. Let us know if you want this feature!`
-      )
-      process.exit(1)
-    }
-
-    if (transactionType === 'CREATE') {
-      console.error(
-        `Sphinx does not support the 'CREATE' opcode, i.e. 'new MyContract(...)'. Please use CREATE2 or CREATE3 instead.`
-      )
-      process.exit(1)
-    } else {
-      if (!transaction.to) {
-        throw new Error(
-          `Transaction does not have the 'to' field. Should never happen.`
-        )
-      }
-
-      const to = ethers.getAddress(transaction.to)
-      if (transactionType === 'CREATE2') {
-        if (to !== DETERMINISTIC_DEPLOYMENT_PROXY_ADDRESS) {
-          console.error(
-            `Detected unsupported CREATE2 factory. Please use the standard factory at: 0x4e59b44847b379578588920cA78FbF26c0B4956C`
-          )
-          process.exit(1)
-        }
-
-        if (!transaction.data || !transaction.gas) {
-          throw new Error(
-            `CREATE2 transaction is missing field(s). Should never happen.`
-          )
-        }
-
-        const salt = ethers.dataSlice(transaction.data, 0, 32)
-        const initCodeWithArgs = ethers.dataSlice(transaction.data, 32)
-        const create2Address = ethers.getCreate2Address(
-          to,
-          salt,
-          ethers.keccak256(initCodeWithArgs)
-        )
-
-        const rawCreate2: RawCreate2ActionInput = {
-          to,
-          create2Address,
-          value: transaction.value ?? '0x0',
-          operation: Operation.Call,
-          txData: transaction.data,
-          initCodeWithArgs,
-          actionType: SphinxActionType.CALL.toString(),
-          gas: transaction.gas,
-          additionalContracts,
-          requireSuccess: deploymentInfo.requireSuccess,
-        }
-        actionInputs.push(rawCreate2)
-      } else if (transactionType === 'CALL') {
-        if (!transaction.data || !transaction.gas) {
-          throw new Error(
-            `CALL transaction is missing field(s). Should never happen.`
-          )
-        }
-
-        const rawCall: RawFunctionCallActionInput = {
-          actionType: SphinxActionType.CALL.toString(),
-          to,
-          value: transaction.value ?? '0x0',
-          txData: transaction.data,
-          operation: Operation.Call,
-          gas: transaction.gas,
-          additionalContracts,
-          requireSuccess: deploymentInfo.requireSuccess,
-        }
-
-        actionInputs.push(rawCall)
-      } else {
-        throw new Error(`Unknown transaction type: ${transactionType}.`)
-      }
-    }
-  }
-
-  return actionInputs
-}
-
 export const makeParsedConfig = (
```

Comment:
I recommend just reading the new `makeParsedConfig` function instead of trying to decipher the diff between the old and new version